src/include/platform/ConfigurationManager.h

```diff
@@ -79,6 +79,13 @@ class ConfigurationManager
     CHIP_ERROR GetDeviceCertificate(uint8_t * buf, size_t bufSize, size_t & certLen);
     CHIP_ERROR GetDeviceIntermediateCACerts(uint8_t * buf, size_t bufSize, size_t & certsLen);
     CHIP_ERROR GetDevicePrivateKey(uint8_t * buf, size_t bufSize, size_t & keyLen);
+    CHIP_ERROR GetDeviceDACCertificate(uint8_t * buf, size_t bufSize, size_t & dacLen);
```

Comment:
I know the existing APIs do this, but it might be better to have new additions look like:
```suggestion
    CHIP_ERROR GetDeviceDACCertificate(MutableByteSpan & buf);
```
Similar for other things here that have these sorts of 3-argument outparams.